hepcrawl/parsers/arxiv.py

```diff
@@ -355,7 +414,9 @@ def authors_and_collaborations(self):
             self._authors_and_collaborations = self._get_authors_and_collaborations(self.root)
         return self._authors_and_collaborations
 
-    @staticmethod
-    def escape_latex_to_unicode(latex_string):
-        l2t = LatexNodes2Text(math_mode="verbatim")
-        return l2t.latex_to_text(latex_string)
+    @classmethod
+    def latex_to_unicode(cls, latex_string):
+        try:
+            return cls._l2t.latex_to_text(latex_string)
+        except Exception:
```

Comment:
not sure what would trigger an exception here. I didn't manage to trigger one.
might be interesting to log it somehow, should it happen